src/SmartComponents/BaselinesTable/BaselinesTable.js

```diff
@@ -39,6 +38,14 @@ class BaselinesTable extends Component {
         this.fetchBaselines();
     }
 
+    componentDidUpdate(prevProps) {
+        const { fetchBaselines } = this.props;
+
+        if (this.props.createBaselineModalOpened === false && prevProps.createBaselineModalOpened === true) {
+            fetchBaselines();
+        }
```

Comment:
What case is this handling? I've not seen us use `componentDidUpdate` before.

Reply:
Found this bug. If you try to clone a baseline from another baseline by selecting a baseline, and then exiting the modal without actually cloning, then the baseline would be selected in the baseline table. So this checks to see if the previous state of the modal was opened and that it is now closed. If so, then it does another fresh fetch and deselects the baselines.